sensorprocess/sensorprocess_test.go

```diff
@@ -595,6 +596,8 @@ func onlineModeIMUTestHelper(
 
 	// set lidar data rate to signify that we are in online mode
 	config.LidarDataRateMsec = 10
+	config.currentLidarData.time = time.Now().UTC().Add(-10 * time.Second)
```

Comment:
[note] `firstLidarReadingTime` and `currentLidarData.time` are used to determine when to allow imu data to be ingested. Both `currentLidarData.time` and `firstLidarReadingTime` must be before the imu time of `time.Now()`